File: eclipse/src/clases_objetos/FacturaClase.java
package clases_objetos;

public class FacturaClase {
	
	//Ingredientes
	public String nombre="";
	public Integer cantidad=0;
	public Double precio=0.0;
	
	//Resultado
	public Double total=0.0;
	
	//FUNCION Procesar
	public void calcularTotal() {
		this.total = this.cantidad * this.precio;
	}
}

File: vscode/src_javascript/clases_objetos/FacturaClase.js
class FacturaClase {
	
	//Ingredientes
	nombre="";
	cantidad=0;
	precio=0.0;
	
	//Resultado
	total=0.0;
	
	//FUNCION Procesar
	calcularTotal() {
		this.total = this.cantidad * this.precio;
	}
}




let factura1 = new FacturaClase();

//Ingredientes
factura1.nombre="Mouse";
factura1.cantidad=5;
factura1.precio=2.5;

//Procesar
factura1.calcularTotal();

console.log("Nombre= " + factura1.nombre + ", Cantidad= "+factura1.cantidad + ", Precio= "+factura1.precio + ", Total="+factura1.total);